metadata-ingestion/src/datahub/ingestion/source/s3/source.py

```diff
@@ -237,7 +238,6 @@ def __init__(self, config: DataLakeSourceConfig, ctx: PipelineContext):
             "data_lake_config",
             config_report,
         )
-
```

Comment:
nit: let's keep this line so there's less changes overall, makes us keeping in sync with the fork more easily.